app/src/androidTest/java/ch/epfl/sweng/studyup/UtilsTest.java

```diff
@@ -0,0 +1,52 @@
+package ch.epfl.sweng.studyup;
```

Comment:
Aren't those some UnitTest and not intrumentationTests?

Reply:
Yes, the same goes for ItemsTest